settings.py.dist

```diff
@@ -10,57 +12,57 @@ ADMINS = (
 
 MANAGERS = ADMINS
 
-DATABASE_ENGINE = 'mysql'           # 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
-DATABASE_NAME = 'master_naturalist_db'             # Or path to database file if sqlite3.
-DATABASE_USER = 'root'             # Not used with sqlite3.
-DATABASE_PASSWORD = 'scott'         # Not used with sqlite3.
-DATABASE_HOST = ''             # Set to empty string for localhost.
-DATABASE_PORT = ''             # Set to empty string for default.
+DATABASE_ENGINE = config('DATABASE_ENGINE', default='mysql') # 'postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
+DATABASE_NAME = config('DATABASE_NAME', default='master_naturalist_db')             # Or path to database file if sqlite3.
+DATABASE_USER = config('DATABASE_USER', default='root')             # Not used with sqlite3.
+DATABASE_PASSWORD = config('DATABASE_PASSWORD', default='scott')         # Not used with sqlite3.
+DATABASE_HOST = config('DATABASE_HOST', default='')             # Set to empty string for localhost.
+DATABASE_PORT = config('DATABASE_PORT', default='')             # Set to empty string for default.
 
 # prefix used for the site.  ie. http://myhost.com/<SITE_ROOT>/
 # for the django standalone server this should be ""
 # for apache this is the url the site is mapped to, probably /pgd
-SITE_ROOT = ''
+SITE_ROOT = config('SITE_ROOT', default='')
 
 # absolute path to the docroot of this site
-DOC_ROOT = ''
+DOC_ROOT = config('DOC_ROOT', default='')
 
 # Local time zone for this installation. Choices can be found here:
 # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
 # although not all choices may be available on all operating systems.
 # If running in a Windows environment this must be set to the same as your
 # system time zone.
-TIME_ZONE = 'America/Chicago'
+TIME_ZONE = config('TIME_ZONE', default='America/Chicago')
 
 # Language code for this installation. All choices can be found here:
 # http://www.i18nguy.com/unicode/language-identifiers.html
-LANGUAGE_CODE = 'en-us'
+LANGUAGE_CODE = config('LANGUAGE_CODE', default='en-us')
 
-SITE_ID = 1
+SITE_ID = config('SITE_ID', default=1, cast=int)
 
 #Google analytics ID. Enter the full id, as in: UA-xxxxxx-x 
-GOOGLE_ID = 'UA-9880071-1'
+GOOGLE_ID = config('GOOGLE_ID', default='UA-9880071-1')
```

Comment:
Use a default value of 'None' instead of that string, since that's our official string and should not be in the repo. :-)